corehq/apps/toggle_ui/utils.py

```diff
@@ -1,7 +1,106 @@
+from couchexport.writers import Excel2007ExportWriter
+from django.utils.translation import ugettext as _
+from io import BytesIO
 from corehq.toggles import all_toggles
 
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.es import UserES
+
 
 def find_static_toggle(slug):
     for toggle in all_toggles():
         if toggle.slug == slug:
             return toggle
+
+
+def get_subscription_info(domain):
+    subscription = Subscription.get_active_subscription_by_domain(domain)
+    if subscription:
+        return subscription.service_type, subscription.plan_version.plan.name
+    else:
+        return None, None
+
+
+def has_dimagi_user(domain):
+    return UserES().web_users().domain(domain).search_string_query('@dimagi.com').count()
```

Comment:
can this not be done in bulk to avoid doing one query per domain? I think you could do this:
```python
user_counts = (
    UserES()
    .web_users()
    .domain(domains)
    .search_string_query('@dimagi.com')
    .terms_aggregation("domain.exact", "domain")
    .size(0)
    .run().aggregations.domain.buckets_list
domains_with_dimag_users = {x.key for x in user_counts if x.doc_count}
```

Alternately, we should at least cache the result since there are going to be common domains in the FFs.